apps/challenges/models.py

```diff
@@ -347,6 +359,16 @@ def save(self, *args, **kwargs):
         return challenge_phase_instance
 
 
+@receiver(signals.post_save, sender="challenges.ChallengePhase")
+def challenge_phase_sync(sender, instance, created, **kwargs):
```

Comment:
change to `challenge_phase_details_sync`

Reply:
Updated.